### DISCUSSION TITLE
Not Injecting

### DISCUSSION THREAD

**Discussion by Unk88** [winstxnhdw/lc-hax#459] [Q&A]
Its not working for me this is the error log I get 
![image_2024-03-11_015304695](https://github.com/winstxnhdw/lc-hax/assets/149540735/25e3ed17-95f2-4ee2-a3f5-661327e13d33)

**Comment by Unk88** [comment]
Resolved: I had steam running games as admin

**Reply by winstxnhdw** [reply]
Oh this is interesting. I have never met anyone who has run Steam games as admin before. I've seen these logs a few times, they are probably doing the same thing as you.

**Reply by Unk88** [reply]
yea I had my steam app set to start as admin it so it started all my games as admin any sharpmono injector could not get perms to open handle is my assumption
